kube/services/revproxy/00nginx-config.yaml

```diff
@@ -102,47 +174,43 @@ data:
           }
           
           location / {
-              set $portal_service http://portal-service.default.svc.cluster.local;
-              if ($cookie_portal_service ~ ^[\w-]+$) {
```

Comment:
why is this removed?

Reply:
I want to replace with changes so we can deploy the whole commons into a namespace rather than just do the portal-switch with a cookie thing - which I don't think has turned out to be too useful ...?